docker.mk

```diff
@@ -82,6 +82,15 @@ drush:
 logs:
 	@docker-compose logs -f $(filter-out $@,$(MAKECMDGOALS))
 
+## debug	:	Enable xdebug.
+.PHONY: debug
+debug:
+	@echo "Starting up containers for $(PROJECT_NAME) with xdebug enabled."
+	@echo "¡¡CAUTION!! X-debug will only work if you have correctly configured docker-compose.xdebug.override.yml file."
+	@docker-compose stop
+	docker-compose pull
```

Comment:
```suggestion
```
I think that this command will be run when the main containers are already up, so pulling should be necessary. Knowing that, do you think pulling the containers may be necessary for debugging?

Reply:
I think that pulling here can cover the case where the developer make a setup and immediately launch make start_dev. It only takes a second to make a pull, so it's not an expensive "just in case".